### DISCUSSION TITLE
Why are the window borders transparent, but the title is not?

### DISCUSSION THREAD

**Discussion by nequals** [tobychui/arozos#126] [General]
Title says it all. It doesn't look good.
Anybody else having the same thought?

**Comment by tobychui** [comment]
We are working with a local design studio to bring new updates to the window designs. It has been released to our devs as nightly build and I hope it will be releasing with v2.20 on Github
![圖片](https://user-images.githubusercontent.com/24617523/232273358-79a6486d-356a-4f28-af5d-c623319a9bc5.png)

**Comment by nequals** [comment]
Discussion closed due to issue being fixed.
